lib/daylite/electronic_address.rb

```diff
@@ -0,0 +1,10 @@
+class Daylite::ElectronicAddress < Daylite::Base
```

Comment:
Here is an example of a model which uses CamelCase notation, even though it's underlying table does not.